src/components/member/tabs/contracts-tab/agreement/CreateQuoteFromAgreement.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { Button } from '../../../../../../shared/hedvig-ui/button'
+import { ThirdLevelHeadline } from '../../../../../../shared/hedvig-ui/typography'
+import { Agreement, Contract } from '../../../../../api/generated/graphql'
+import {
+  createQuoteFromAgreementOptions,
+  useCreateQuoteFromAgreement,
+} from '../../../../../graphql/use-create-quote'
+import { useQuotes } from '../../../../../graphql/use-quotes'
+import { Notification } from '../../../../../store/actions/notificationsActions'
+
+export const CreateQuoteFromAgreement: React.FunctionComponent<{
+  agreement: Agreement
+  contract: Contract
+  showNotification: (data: Notification) => void
+}> = ({ agreement, contract, showNotification }) => {
+  const [createQuote] = useCreateQuoteFromAgreement(contract)
+  const [quotes, loadingQuotes] = useQuotes(contract.holderMemberId)
+
+  return (
+    <>
+      <ThirdLevelHeadline>Create Quote</ThirdLevelHeadline>
+      <>
+        {quotes
+          .map((quote) => quote.originatingProductId)
```

Comment:
Even though it worked like this before. We should allow creating multiple quotes from an agreement IF that quote has been activated/removed/whatever. So we should add something like this:
```suggestion
        {quotes
          .filter((quote) => !quote.isActivated)
          .map((quote) => quote.originatingProductId)
```